src/styles/_common/_colors.scss

```diff
@@ -80,6 +80,12 @@ $palette: (
     'error': #ff6b6b,
     'info': #00adef,
     'text': #fff
+  ),
+  'footer': (
+    'background': #fafafa,
```

Comment:
I get your point, but in this file, everything is a color. If we have to add "color" to all the variables we would have things like "errorColor", "infoColor", "textColor". I think that in this context, as the function is already named "color" it would be redundant.
Remebering that the usage is example:

```
.footer {
   background-color: color('footer', 'background');
}
```
@mmelo , what do you think about it?